customize_image.sh

```diff
@@ -344,7 +337,6 @@ chroot "$IMAGEDIR" systemctl disable hciuart
 # disable 3rd party software
 if [ "$MINIMG" != "1" ]; then
 	chroot "$IMAGEDIR" systemctl disable logiclab
-	chroot "$IMAGEDIR" systemctl disable nodered
```

Comment:
I assume the upstream installer enabled the script by default and since we're not using it anymore we don't have to disable it?

Reply:
No, we only copy the systemD unit file. There is no installer that sets the unit file to enabled.